docs/index.md

```diff
@@ -33,8 +33,32 @@ vagrant init DrupalMel/beetbox
 vagrant up
 ```
 
-After which you can install the site at http://drupal-8-0-2.local/install.php
+After which you can install the site at [http://drupal-8-0-2.local/install.php](http://drupal-8-0-2.local/install.php)
 
+### Default Database
+
+#### MySQL 
+
+- Hostname: localhost or 127.0.0.1
+- Username: beetbox
+- Password: beetbox
+- Database: beetbox
+
+## Customisation
+
+All customisation is done via a YAML configuration file at `.beetbox/config.yml`. If this file doesn't exist it is automatically created the first time you start up the VM.
+Configuration is applied hierarchically so any attribute set in your project config file will override a lower level default setting.
+ 
+Each ansible role we use has a set of defaults eg. the PHP role -- https://github.com/geerlingguy/ansible-role-php/blob/master/defaults/main.yml 
```

Comment:
Just realised that we might need a way to link to the version of these defaults at the time they were at the last release.